src/renderer/reducers/accounts.js

```diff
@@ -87,6 +92,29 @@ export const shallowAccountsSelector: OutputSelector<
   Account[],
 > = shallowAccountsSelectorCreator(accountsSelector, a => a);
 
+export const subAccountByCurrencyOrderedSelector: OutputSelector<
+  State,
+  { currency: CryptoCurrency | TokenCurrency },
+  (AccountLike & { parentAccount?: Account })[],
```

Comment:
the returned type is a bit unconventional (`AccountLike & {parentAccount}`)

This would be better to have a `{ parentAccount: ?Account, account: AccountLike }`  which is convention everywhere else.

This is especially important to preserve account reference and therefore react performance

Reply:
done

Reply:
Oh well... not done at all... it changes SO MANY THINGS